applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/naca0012_small_sensitivities_adjoint_analytical_parameters.json

```diff
@@ -19,7 +19,7 @@
         "compute_reactions"			   : false,
         "response_function_settings" : {
                 "response_type"     : "adjoint_lift_jump_coordinates",
-                "use_kratos"        : true,
+                "analyzer"          : "kratos",
```

Comment:
This is from a test case, that should not go on the master, am I write?

Reply:
The two files in the CompressiblePotentialFlowApplication/tests where already there. I just updated them. @inigolcanalejo should they be on the repo / are they in use? I don't remember.

Reply:
anyway i would not want to remove them in this MR because its unrelated.

Reply:
Alright. Did not realize it was in the compressible flow app.

Reply:
Yes, they are in used and they should be in the repo. Even though they are called naca0012 it is actually a small test with a rhombus geometry. We need to change the names...

Reply:
Is this change ok for you?

Reply:
Yes. @marcnunezc is this change also ok for you?

Reply:
Yes I am fine with the change!